mast_aladin_lite/tests/test_overlays_dict.py

```diff
@@ -0,0 +1,404 @@
+from astropy.table import Table
+from astropy.coordinates import SkyCoord, Angle
+import astropy.units as u
+import pytest
+from unittest.mock import Mock
+import warnings
+import re
+
+from mast_aladin_lite import MastAladin
+from mast_aladin_lite.mast_overlay import MastOverlay, MastOverlayType
+from ipyaladin.elements.error_shape import EllipseError, CircleError
+from regions import CircleSkyRegion
+from ipyaladin import Marker
+
+
+mast_aladin = MastAladin()
+
+
```

Comment:
suggestion: this test is covering a lot of different use cases, which I have found makes tracking down errors potentially much more time consuming. This is because fixing one use case doesn't necessarily fix them all and you run into a cat an mouse game where you fix one issue, then rerun the test and fix the next, and so on until they all work.

If the tests are instead broken out into individual tests, that each only covers one use case, you'll get all of the errors at once and can address then in one revision. This is really important as our tests balloon in execution time (see jdaviz)